CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change log
 -----------
 
+# v2.0.4+rev2 - 2017-05-28
```

Comment:
Yeah, but until now we did a release when we bumped meta-resin. I thought I'd follow the same convention